src/scripts/destinyTrackerApi/reviewDataCache.js

```diff
@@ -0,0 +1,86 @@
+import _ from 'underscore';
+import { itemTransformer } from './itemTransformer.js';
+
+class reviewDataCache {
```

Comment:
We should probably cache these in an IndexedDB table instead of just in memory. I've been meaning to import the `idb` package so we can start using separate tables (right now I use idb-keyval which puts it all in the same place).